client_test/directfund_simple_tcp_test.go

```diff
@@ -0,0 +1,46 @@
+// Package client_test contains helpers and integration tests for go-nitro clients
+package client_test // import "github.com/statechannels/go-nitro/client_test"
+
+import (
+	"io"
+	"testing"
+	"time"
+
+	"github.com/statechannels/go-nitro/client"
+	"github.com/statechannels/go-nitro/client/engine/chainservice"
+	simpletcp "github.com/statechannels/go-nitro/client/engine/messageservice/simple-tcp"
+	"github.com/statechannels/go-nitro/client/engine/store"
+	"github.com/statechannels/go-nitro/crypto"
+	"github.com/statechannels/go-nitro/types"
+)
+
+// setupClientWithSimpleTCP is a helper function that contructs a client and returns the new client and its store.
+func setupClientWithSimpleTCP(pk []byte, chain chainservice.MockChain, peers map[types.Address]string, logDestination io.Writer, meanMessageDelay time.Duration) (client.Client, *simpletcp.SimpleTCPMessageService) {
+	myAddress := crypto.GetAddressFromSecretKeyBytes(pk)
+	chain.Subscribe(myAddress)
```

Comment:
I think you don't need these calls since #687

Reply:
https://github.com/statechannels/go-nitro/pull/681/commits/d03a68834a5f5ab76f6c05e52bd3a3834d86cc7a